charts/tidb-cluster/values.yaml

```diff
@@ -77,6 +77,7 @@ pd:
   # we can only specify clusterIP and loadBalancerIP now
   service:
     clusterIP: "None"
+    # portName: "<the default name will be used>"
```

Comment:
can you replace `<the default name will be used>` with the actual port name used for pd? I guess it's `client` here?